sidebars.js

```diff
@@ -288,6 +288,17 @@ const sidebars = {
           dirName: 'nodes/validate'
         }
       ]
+    },
+    {
+      type: 'category',
+      label: 'FAQ',
+      collapsed: false,
+      items: [
+        {
+          type: 'autogenerated',
+          dirName: 'nodes/FAQ'
```

Comment:
FAQ is not a directory, it is a file only.

Reply:
I left it open to possibly add more so if the FAQ grows we can add future files, but open to whatever works best in the short term.

Reply:
I don't believe so as of yet.